### ISSUE TITLE
Reduce download size of initial clone

### ISSUE THREAD

**Issue by neonichu** [contentful/contentful.objc#24]
A bunch of stuff is not needed for many use-cases:

- *Pods* directory could be a submodule (18MB)
- Recordings for tests could be a submodule (11MB)
- Binary framework is already on S3 and GH releases (14MB)

**Comment by neonichu** [comment]
```
$ git clone https://github.com/contentful/contentful.objc.git --single-branch --depth 1
```

is now 2MB :tada:

**Comment by neonichu** [comment]
Full clone is 20MB, Pods and binary frameworks have been purged from the history. It's quite some work to remove the fixtures and recordings as they were spread out, so this should be enough for now.

**Comment by neonichu** [comment]
Installing the latest version into a project via CP takes less than half the time now \o/

Before: 0m25.365s
After: 0m11.861s
